prime-router/build.gradle.kts

```diff
@@ -27,7 +27,7 @@ import org.jooq.meta.jaxb.ForcedType
 import java.io.FileInputStream
 import java.time.LocalDateTime
 import java.time.format.DateTimeFormatter
-import java.util.Properties
+import java.util.*
```

Comment:
It seems like your formatter settings might be a bit off, under `Settings`->`Editor`->`File Types`->`Kotlin`, can you make sure you have `*.kt` and `*.kts` and then reformat this file? It should revert some of the formatting changes that we're seeing here.

Reply:
fixed, last time did not trigger a save on the .kts...